Test/Core/DualityTests.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
This file shouldn't be checked in.

Reply:
Why not? It doesnt hurt ppl without resharper and it will only help (in this case) to keep the namespaces correct.

Reply:
It's a third party tools settings file: Not helpful in regular VS, not required for the build, and whether or not someone uses Resharper is a matter of personal preferences. People who use that tool can still have that file locally.